ontopy/utils.py

```diff
@@ -447,16 +447,16 @@ def _validate_installed_version(
     """
     # pylint: disable=import-outside-toplevel
     import importlib
-    from packaging.version import parse as parse_version, LegacyVersion, Version
+    from packaging.version import parse as parse_version, Version
 
     if isinstance(min_version, str):
         min_version = parse_version(min_version)
-    elif isinstance(min_version, (LegacyVersion, Version)):
+    elif isinstance(min_version, (Version)):
```

Comment:
```suggestion
    elif isinstance(min_version, Version):
```
This is not going to work if `Version` is only imported in the `TYPE_CHECKING` block!